### DISCUSSION TITLE
Instructions Still Relevant for Bookworm OS?

### DISCUSSION THREAD

**Discussion by KDMcMullan** [ct-Open-Source/tuya-convert#1112] [Q&A]
The new Bookwork OS doesn't seem to use wpa_supplicant.conf.
Will the guide still work if folks leave out the section about deleting the WiFi setup from that file?

**Comment by boldranet** [comment]
Tried it - my first problems were conflicts between pip wanting to install paho-mqtt, sslpsk and pycryptodomex and apt wanting to manage python packages. I got around that with `sudo pip3 install sslpsk  --break-system-packages` (for each of those packages).

It then got all the way to "Starting AP in a screen" but no AP ever appeared, so your answer seems to be _no_
